src/packages/@app/pages/Auth/entry-point/EntryPoint.svelte

```diff
@@ -145,14 +143,16 @@
       </div>
     </div>
     <div
-      class="mb-2 mt-2"
-      style="text-align:center; font-size:12px; color:var(  --text-secondary-350);"
+      class="mb-2 mt-2 w-[266px]"
+      style="text-align:center; font-size:12px; color:var(  --text-secondary-350); padding: 12px 0px ; width: 266px;
```

Comment:
Are we sure hard coding width of something is a good idea here? We need to remember this needs to be responsive. We are talking about a desktop application here.